### PR TITLE
Adding lookbook links to docs output

### PR DESCRIPTION
**Pull request by jonrohan**
image

- The links will only show up if the component has a preview written.
- I had to suffix the url with `/default/` which assumes all components have a `def default` preview. I did this because leaving it off rails ends up redirecting to the azure server url example https://primer.style/view-components/lookbook/inspect/primer/beta/avatar/

### CODE REVIEW

File: lib/tasks/docs.rake [hunk 1]
```diff
@@ -485,6 +487,18 @@ namespace :docs do
     "https://primer.style/view-components/stories/?path=/story/#{path}"
   end
 
+  def lookbook_url(component)
+    path = component.name.underscore
+
+    "https://primer.style/view-components/lookbook/inspect/#{path}/default/"
```
Comment (langermank) [inline lib/tasks/docs.rake:493]: Does this mean that there needs to be a preview called "default"?

Comment (jonrohan) [reply lib/tasks/docs.rake:493]: I don't mind what we call it, but we do need to make sure they all have the same method

Comment (langermank) [reply lib/tasks/docs.rake:493]: I started an ADR for preview/story format and recommend that every component has a "default" so I like this 😄

**Review by langermank — APPROVED** [review]

File: lib/tasks/docs.rake (hunk 1 above)
Comment (joelhawksley) [reply lib/tasks/docs.rake:493]: @jonrohan perhaps we could add a linter or test that makes sure each component has a preview called `default`, then?

**Pull request merged**